packages/components/src/components/flipcard/flipcard.stories.ts

```diff
@@ -50,281 +51,310 @@ export const Default = {
+        style="aspect-ratio:3/4;"
+        class="mb-12"
+      >
+        <p slot="front" class="slot slot--border slot--text h-12 w-full">Front slot</p>
+        <p slot="back" class="slot slot--border slot--text h-12 w-full">Back slot</p>
+        <img
+          slot="media-front"
+          class="object-cover h-full w-full"
+          src="./placeholders/images/generic.jpg"
+          alt="Generic"
+        />
+        <img
+          slot="media-back"
+          class="object-cover h-full w-full"
+          src="./placeholders/images/generic.jpg"
+          alt="Generic"
+        />
+      </sd-flipcard>
+      <sd-flipcard
+        activation="click hover"
+        flip-direction="horizontal"
+        front-variant="primary-100"
+        back-variant="primary-100"
+        style="aspect-ratio:16/9;"
+      >
+        <p slot="front" class="slot slot--border slot--text h-12 w-full">Front slot</p>
+        <p slot="back" class="slot slot--border slot--text h-12 w-full">Back slot</p>
+        <img
+          slot="media-front"
+          class="object-cover h-full w-full"
+          src="./placeholders/images/generic.jpg"
+          alt="Generic"
+        />
+        <img
+          slot="media-back"
+          class="object-cover h-full w-full"
+          src="./placeholders/images/generic.jpg"
+          alt="Generic"
+        />
+      </sd-flipcard>
+    </div>`
 };
 
 /**
- * Here is a sample of the `sd-flipcard` with custom content in the `front` and `back` slots. The activation is set to `click-only` in order allow the user to click on links/buttons inside the flipcard.
+ * Use the `activation` attribute to determine the activation type of the flipcard.
+ * - `click hover` (default)
+ * - `click`
  */
 
-export const Sample = {
-  name: 'Sample: Custom Content',
-  render: () => {
-    return html`
-      <sd-flipcard activation="click" front-variant="primary" back-variant="primary-100">
-        <div class="py-4 px-6" slot="front">
-          <h4 class="sd-headline sd-headline--inline sd-headline--size-lg sd-headline--inverted mb-2">
-            <sd-icon name="content/picture" library="global-resources"></sd-icon>
-            Nisi eu excepteur anim esse
-          </h4>
-
-          <p class="sd-paragraph text-left sd-paragraph--inverted">
-            Lorem ipsum dolor sit amet per niente da faremmasds nonnummy dolore lorem ipsum dolor sit amet consectuer
-          </p>
-        </div>
-
-        <div class="py-4 px-6" slot="back">
-          <h4 class="sd-headline sd-headline--inline sd-headline--size-lg mb-2">
-            <sd-icon name="content/picture" library="global-resources"></sd-icon>
-            Nisi eu excepteur anim esse
-          </h4>
-
-          <p class="sd-paragraph text-left mb-4">
-            Lorem ipsum dolor sit amet per niente da faremmasds nonnummy dolore lorem ipsum dolor sit amet consectuer
-          </p>
+export const Activation = {
+  render: () =>
+    html` <div class="grid grid-cols-2">
```

Comment:
Same as before, use `gap-*` and remove divs around the component

Reply:
Updated